proto/cdcpb.proto

```diff
@@ -46,6 +53,7 @@ message Error {
     Compatibility compatibility = 5;
     ClusterIDMismatch cluster_id_mismatch = 6;
     errorpb.ServerIsBusy server_is_busy = 7;
+    Congested congested = 8;
```

Comment:
How does old TiCDC handle this error?

Reply:
The old TiCDC treat this congested error as unknown error, print logs.